### DISCUSSION TITLE
Pack API Support: Questions & Answers + Ideas

### DISCUSSION THREAD

**Discussion by chamrun** [tapsi-pack/api-doc#8] [Ideas]
Hey there Pack API Users!

We've set up this GitHub Discussion as a space to streamline our support process. Here, we can gather FAQs, share ideas, and collaborate on improving your experience with our API.

Your input is crucial, so dive in, ask questions, and pitch in your suggestions. Let's work together to make Pack API support even better!

Excited to hear your thoughts!

**Comment by plus98** [comment]
من دارم API شما رو برای یه فروشگاه اینترنتی پیاده می‌کنم. اینا روی دیوارم فروشگاه دارن و از اون طریق محموله ۴۵ هزار تومانی می ساختن. برای شبیه‌سازی اون حالت هیچ امکانی از طریق API شما وجود نداره؟
